src/configure/helper/devOps/azureDevOpsHelper.ts

```diff
@@ -117,7 +117,7 @@ export class AzureDevOpsHelper {
             },
             process: {
                 type: 2,
-                yamlFileName: path.join(inputs.pipelineConfiguration.workingDirectory, path.basename(inputs.pipelineConfiguration.filePath))
+                yamlFileName: path.basename(inputs.pipelineConfiguration.filePath)
```

Comment:
logic here and while detecting file path to commit is different because of which this bug has occurred. I think better option is inputs.pipelineConfiguration.filePath should contain the filePath in w.r.t. to root of repository. And join local rootDirectoryPath to inputs.pipelineConfiguration.filePath while adding file to localGit.

Reply:
As discussed, since the requirement is to have azure pipeline files always at the root level, keeping it this way and adding a comment in the code for better understanding.